metadata/ComputerVision__3__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2022.04.07001",
+  "cpu": 0,
+  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON36_CV",
+  "name": "ComputerVision",
+  "outputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Computer Vision",
+  "projectDescription": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
```

Comment:
Shouldn't this be 22.4?

Reply:
The model should work with the 21.10 AIC as well, as per this conversation https://uipath-product.slack.com/archives/CN7P410DD/p1646142714147669